engine/evd.mli

```diff
@@ -655,7 +655,7 @@ val update_sigma_env : evar_map -> env -> evar_map
 
 val fresh_sort_in_family : ?loc:Loc.t -> ?rigid:rigid -> evar_map -> Sorts.family -> evar_map * Sorts.t
 val fresh_constant_instance : ?loc:Loc.t -> env -> evar_map -> Constant.t -> evar_map * pconstant
-val fresh_inductive_instance : ?loc:Loc.t -> env -> evar_map -> inductive -> evar_map * pinductive
+val fresh_inductive_instance : ?loc:Loc.t -> ?rigid:rigid -> env -> evar_map -> inductive -> evar_map * pinductive
```

Comment:
Not sure if I should bother adding the parameter to the other functions here.

Reply:
For uniformity, I think it'd be good.

Reply:
(If you do this tweak fast, I'll merge the PR right after the CI completes.)